source/PlayState.hx

```diff
@@ -3345,15 +3350,16 @@ class PlayState extends MusicBeatState
 				gfSpeed = value;
 
 			case 'Philly Glow':
-				var lightId:Int = Std.parseInt(value1);
-				if(Math.isNaN(lightId)) lightId = 0;
-
-				var doFlash:Void->Void = function() {
-					var color:FlxColor = FlxColor.WHITE;
-					if(!ClientPrefs.flashing) color.alphaFloat = 0.5;
-
-					FlxG.camera.flash(color, 0.15, null, true);
-				};
+				if (WeekData.getWeekFileName() == 'week3' && curStage == 'philly') {
```

Comment:
I'm just gonna comment on each of these lines.

Reply:
I'm just gonna take your vital organs